### ISSUE TITLE
directPrintPdf on Mac uses wrong printer

### ISSUE THREAD

**Issue by lehnen** [DavBfr/dart_pdf#879]
**Describe the bug**
On my Mac (Monterey 12.0.1) I have three printers installed. No matter what printer I select (programmatically or by using pickPrinter, directPrintPdf always uses the second printer in my list.

**To Reproduce**
Code snippet to reproduce the behavior:
PicPrinter shows my three printers (Epson Ink / Postscript Laser / PDFwriter). After choosing a printer, the print('Selected printer...) shows the correct selected one. But every time directPrintPdf prints the document on the Postscript Laser.
```dart
var pdfBinary = await Printing.convertHtml(format: format, html: html);
final Printer? printer = await Printing.pickPrinter(context: context);
print('Selected printer: $printer');
if (printer != null) {
  Printing.directPrintPdf(printer: printer, onLayout: (format) => pdfBinary);
}
```

**Expected behavior**
directPrintPdf should use the selected printer.

**Screenshots**
would not be helpfull, I think.

**Flutter Doctor**
[✓] Flutter (Channel stable, 2.8.0, on macOS 12.0.1 21A559 darwin-x64, locale de-DE)
    • Flutter version 2.8.0 at /Users/lehnen/Documents/Daten/daten/Flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision cf44000065 (5 days ago), 2021-12-08 14:06:50 -0800
    • Engine revision 40a99c5951
    • Dart version 2.15.0

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/lehnen/Library/Android/Sdk
    • Platform android-30, build-tools 30.0.3
    • ANDROID_HOME = /Users/lehnen/Library/Android/Sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-x64     • macOS 12.0.1 21A559 darwin-x64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 96.0.4664.93

• No issues found!
```
```

**Desktop (please complete the following information):**
 - [ ] iOS
 - [ ] Android
 - [ ] Browser
 - [ ] Windows
 - [ ] Linux
 - [x] Mac

**Additional context**

**Comment by DavBfr** [comment]
Fixed in the master branch (printing-5.6.8)

**Comment by lehnen** [comment]
Hey DavBfr, 

I am sorry for contacting you again on this issue, but there is still the 5.6.6 on pub and I don't know, how to update to 5.6.8, which I need to proceed my development. Is there another way to add the repository to my project than pub.dev?

Thx for your answer,
Volker
